.buildkite/premerge.steps.yaml

```diff
@@ -22,6 +22,18 @@ common: &common
   plugins:
     - ca-johnson/taskkill#v4.1: ~
 
+script_runner: &script_runner
+  agents:
+  - "agent_count=8"
+  - "capable_of_building=platform"
+  - "environment=production"
+  - "machine_type=quarter"
+  - "permission_set=builder"
+  - "platform=linux"
+  - "queue=${CI_LINUX_BUILDER_QUEUE:-v3-1569277658-42ebcf6cb8969a53-------z}"
```

Comment:
On a similar note, what is the thinking behind having this on separate build agents?

Reply:
This is the general platform build queue, so most of those agents should not be existing because of this pipeline.
This runs as a separate build step so that the version check can fail, but the build steps can still succeed. That way, if you're iterating on a PR that merges into preview/release, you'll know that the change in the version file is the only thing that failed your build.

Reply:
Right, that explains why there are so many agents :)

Do you know if there is any process for us getting notified if platform build queue gets updated?

Reply:
Not that I'm aware of. However since queues are version controlled an update won't break anything, and just for running scripts we shouldn't have pressing needs for changes.